travvy/src/components/Home.vue

```diff
@@ -13,21 +13,26 @@
     <label for="Destination">Select a Destination:</label>
     <!-- select destinations to find attractions in --> 
     <select v-model="city" id="Destination" name="Destination" class="destination">
-      <option value="Toronto">Toronto</option>
-      <option value="New York City">New York City</option>
+      <option value="Toronto">Toronto, Canada</option>
```

Comment:
this is good for stubbing while the backend hasnt been done

Reply:
youre right lucy, testing the rest of the functionality of the home page will be a lot easier until the backend has caught up